### PR TITLE
Delete API key button now visible in light mode

### LINKED ISSUE #7179: Minor bug : Delete button not visible in light mode
> [!Note]
> Under **Developers/API**, when deleting API key delete button is not visible in light mode until entered **yes**
> "test" is not capital whereas every other heading is capital

## Current Behaviour
Screenshot 2024-09-20 at 5 33 03 PM

Screenshot 2024-09-20 at 5 33 23 PM

## Expected behavior

Screenshot 2024-09-20 at 5 36 03 PM

Screenshot 2024-09-20 at 5 35 44 PM
Screenshot 2024-09-20 at 5 37 27 PM

### PR DESCRIPTION
**Pull request by BKM14**
Closes #7179
The font color of the button has been set to use `theme.font.color.danger`. This means that the color will be red regardless of whether the theme is inverted or not, which was the issue before.

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/input/button/components/Button.tsx [hunk 1]
```diff
@@ -283,11 +283,9 @@ const StyledButton = styled('button', {
                       : theme.background.transparent.medium
                   }`
                 : 'none'};
-              opacity: ${disabled ? 0.24 : 1};
```
Comment (Bonapara) [inline packages/twenty-front/src/modules/ui/input/button/components/Button.tsx:286]: Why removing this part @BKM14? Opacity is set to 24% in Figma

CleanShot 2024-10-25 at 10 16 25

**Review by ehconitin — COMMENTED** [review]
Please resolve the conflicts! Thanks

File: packages/twenty-front/src/modules/ui/input/button/components/Button.tsx (hunk 1 above)
Comment (BKM14) [reply packages/twenty-front/src/modules/ui/input/button/components/Button.tsx:286]: Hey, sorry about that. I'm new to this repository and didn't know figma files were available too. I wasn't able to see the borders too well with the button disabled, hence I increased the opacity. I'll change it back to 0.24

Comment (Bonapara) [reply packages/twenty-front/src/modules/ui/input/button/components/Button.tsx:286]: Now worries, thanks! Did you find access to the Figma file? https://www.figma.com/design/xt8O9mFeLl46C5InWwoMrN/Twenty?node-id=34844-84769&t=JmWAfjD0ufScVlpq-1

Comment (BKM14) [reply packages/twenty-front/src/modules/ui/input/button/components/Button.tsx:286]: Yes, I have reverted the opacity to 0.24 and committed the changes.
Please review it.
Thank you.

**Comment by ehconitin** [comment]
There is a conflict, please resolve it for the review.

**Pull request closed without merging**